Description: Removed ordereddict from requirements.txt
 This is messing with some unit tests, so let's just remove it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-18

--- python-glance-store-0.3.0.orig/requirements.txt
+++ python-glance-store-0.3.0/requirements.txt
@@ -12,6 +12,3 @@ eventlet>=0.13.0 #s3
 six>=1.4.1
 
 jsonschema>=2.0.0,<3.0.0
-
-# py2.6 compat
-ordereddict
